clang/test/CodeGen/pr87758.c

```diff
@@ -0,0 +1,28 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --version 4
+// RUN: %clang_cc1 -triple x86_64-linux-gnu -O3 -emit-llvm -o - %s \
```

Comment:
There is no need to use -O3 here to prove what you want, I think.

Reply:
You don't need to put every // CHECK line. Since we are interested only in the flag associated with the calls to the builtins you can just put those line.